src/Symfony/Bridge/Twig/Tests/Validator/Constraints/TwigValidatorTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Validator\Constraints;
+
+use Symfony\Bridge\Twig\Validator\Constraints\Twig;
+use Symfony\Bridge\Twig\Validator\Constraints\TwigValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+use Twig\TwigFilter;
+
+/**
+ * @author Mokhtar Tlili <tlili.mokhtar@gmail.com>
+ */
+class TwigValidatorTest extends ConstraintValidatorTestCase
+{
+    protected function createValidator(): TwigValidator
+    {
+        $environment = new Environment(new ArrayLoader());
+        $environment->addFilter(new TwigFilter('humanize_filter', fn ($v) => $v));
+
+        return new TwigValidator($environment);
+    }
+
+    /**
+     * @dataProvider getValidValues
+     */
+    public function testTwigIsValid($value)
+    {
+        $this->validator->validate($value, new Twig());
+
+        $this->assertNoViolation();
+    }
+
+    /**
+     * @dataProvider getInvalidValues
+     */
+    public function testInvalidValues($value, $message, $line)
+    {
+        $constraint = new Twig('myMessageTest');
+
+        $this->validator->validate($value, $constraint);
+
+        $this->buildViolation('myMessageTest')
+            ->setParameter('{{ error }}', $message)
+            ->setParameter('{{ line }}', $line)
+            ->setCode(Twig::INVALID_TWIG_ERROR)
+            ->assertRaised();
+    }
+
+    public static function getValidValues()
+    {
+        return [
+            ['Hello {{ name }}'],
+            ['{% if condition %}Yes{% else %}No{% endif %}'],
+            ['{# Comment #}'],
+            ['Hello {{ "world" | upper }}'],
```

Comment:
```suggestion
            ['Hello {{ "world"|upper }}'],
```

And everywhere else whare you're using the `|` operator.

Reply:
Done